RNTester/js/TabBarIOSExample.js

```diff
@@ -15,6 +15,7 @@ var {
   TabBarIOS,
   Text,
   View,
+  Button
```

Comment:
since you are here, could you throw a comma at the end of this? :)

Reply:
Done!